vowpalwabbit/cats.cc

```diff
@@ -112,13 +117,34 @@ void reduction_output::output_predictions(std::vector<std::unique_ptr<VW::io::wr
   for (auto& f : predict_file_descriptors) f->write(str.c_str(), str.size());
 }
 
-// "average loss" "since last" "example counter" "example weight"
-// "current label" "current predict" "current features"
-void reduction_output::report_progress(vw& all, const cats&, const example& ec)
+void reduction_output::report_progress(vw& all, const cats& data, const example& ec)
```

Comment:
A unit test around this with  +/-data.min_value, +/-data.max_value, +/-actions would be helpful.

Reply:
Added unit tests